app/config/sources.inc.php

```diff
@@ -588,6 +595,13 @@
         'supported_locales' => $newsletter_locales,
     ],
     'mozorg/plugincheck-redesign.lang' => [
+        'flags'             => [
+          'obsolete'        => ['all'],
+        'supported_locales' => $mozillaorg,
```

Comment:
Check this array and try to figure out why it's wrong, where parenthesis open and close.